packages/ns-plug/files/ns-plug

```diff
@@ -71,7 +71,7 @@ response=""
 until [ "${valid}" -eq 1 ] || [ "${watchdog}" -ge "${max_attempts}" ]
 do
     response=$(curl ${curl_opts} -H "Content-Type: application/json" -H "RegistrationToken: ${token}" ${server}/api/units/register -X POST --data '{"unit_id": "'${unit_id}'", "username": "'${user}'", "password": "'${secret}'", "unit_name": "'${unit_name}'", "version": "'${VERSION_ID}'", "subscription_type": "'${subscription_type}'", "system_id": "'${system_id}'"}')
-    http_code=$(echo ${response} | jq -r .code)
+    http_code=$(echo ${response} | jq -r .code // empty)
```

Comment:
This change prevents the connection if the server is healthy.

Reply:
it just quiet the warning, if the server is up I can connect

Reply:
I've tested it again: it does not connect, the script loops
To fix it, use:
```suggestion
    http_code=$(echo ${response} | jq -r '.code // empty')
```